src/ontology/mondo-edit.obo

```diff
@@ -419371,6 +419431,264 @@ xref: Orphanet:2024 {source="MONDO:subClassOf", source="OMIM:617626"}
+name: cone-rod synaptic disorder syndrome, congenital nonprogressive
+synonym: "CONE-ROD SYNAPTIC DISORDER SYNDROME, CONGENITAL NONPROGRESSIVE; CRSDS" RELATED [OMIM:618970]
+xref: OMIM:618970 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033544
+name: tolchin-le caignec syndrome
+synonym: "Intellectual Developmental Disorder With Behavioral Abnormalities and Variable Bone Defects" RELATED [OMIM:618971]
+synonym: "TOLCHIN-LE CAIGNEC SYNDROME; TOLCAS" RELATED [OMIM:618971]
+xref: OMIM:618971 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033545
+name: mitochondrial dna depletion syndrome 19
+synonym: "MITOCHONDRIAL DNA DEPLETION SYNDROME 19; MTDPS19" RELATED [OMIM:618972]
+xref: OMIM:618972 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+is_a: MONDO:0018158 {source="OMIM:618972"} ! mitochondrial DNA depletion syndrome
+
+[Term]
+id: MONDO:0033546
+name: neurodegeneration, infantile-onset, biotin-responsive
+synonym: "NEURODEGENERATION, INFANTILE-ONSET, BIOTIN-RESPONSIVE; NERIB" RELATED [OMIM:618973]
+synonym: "Smvt Deficiency" RELATED [OMIM:618973]
+synonym: "Sodium-Dependent Multivitamin Transporter Deficiency" RELATED [OMIM:618973]
+xref: OMIM:618973 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033547
+name: li-ghorbani-weisz-hubshman syndrome
+synonym: "LI-GHORBANI-WEISZ-HUBSHMAN SYNDROME; LIGOWS" RELATED [OMIM:618974]
+xref: OMIM:618974 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033548
+name: myopathy, congenital, with diaphragmatic defects, respiratory insufficiency, and dysmorphic facies
+synonym: "Myopathy, Congenital, Due to Myod1 Deficiency" RELATED [OMIM:618975]
+synonym: "MYOPATHY, CONGENITAL, WITH DIAPHRAGMATIC DEFECTS, RESPIRATORY INSUFFICIENCY, AND DYSMORPHIC FACIES; MYODRIF" RELATED [OMIM:618975]
+xref: OMIM:618975 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033549
+name: optic atrophy 12
+synonym: "OPTIC ATROPHY 12; OPA12" RELATED [OMIM:618977]
+xref: OMIM:618977 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003608 {source="OMIM:618977"} ! optic atrophy
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033550
+name: high density lipoprotein cholesterol level quantitative trait locus 7
+synonym: "HIGH DENSITY LIPOPROTEIN CHOLESTEROL LEVEL QUANTITATIVE TRAIT LOCUS 7; HDLCQ7" RELATED [OMIM:618979]
+xref: OMIM:618979 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+
+[Term]
+id: MONDO:0033551
+name: immunodeficiency 72 with autoinflammation
+synonym: "IMMUNODEFICIENCY 72 WITH AUTOINFLAMMATION; IMD72" RELATED [OMIM:618982]
+xref: OMIM:618982 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
+is_a: MONDO:0021094 {source="OMIM:618982"} ! immunodeficiency disease
+
+[Term]
+id: MONDO:0033552
+name: blood group, lewis system
```

Comment:
not a disease. obsolete and add to obsolete.obo

Reply:
I obsoleted 'high density lipoprotein cholesterol level quantitative trait locus 7'